core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -20,7 +20,7 @@ require(
   'components/forms/custom-forms-directives/thumbnail-uploader.directive.ts');
 require('components/skill-selector/select-skill-modal.controller.ts');
 require(
-  'components/skill-selector/skill-selector.directive.ts');
+  'components/skill-selector/skill-selector.component.ts');
```

Comment:
import not required.

Reply:
Going to wait as I'm curious about [this question](https://github.com/oppia/oppia/pull/11791#discussion_r587011631) before making all the changes.

Reply:
Leaving it in as per [this comment](https://github.com/oppia/oppia/pull/11791#discussion_r587057593)